src/containers/mode-tools.jsx

```diff
@@ -255,17 +218,14 @@ class ModeTools extends React.Component {
 }
 
 ModeTools.propTypes = {
-    clearSelectedItems: PropTypes.func.isRequired,
-    clipboardItems: PropTypes.arrayOf(PropTypes.array),
-    format: PropTypes.oneOf(Object.keys(Formats)).isRequired,
-    incrementPasteOffset: PropTypes.func.isRequired,
+    format: PropTypes.oneOf(Object.keys(Formats)),
     mode: PropTypes.oneOf(Object.keys(Modes)),
+    onCopyToClipboard: PropTypes.func.isRequired,
+    onPasteFromClipboard: PropTypes.func.isRequired,
```

Comment:
Since this comes from the `CopyPasteHOC`, then the component exported from this file should be wrapped with `CopyPasteHOC`, and then anything using the component doesn't need to know about the wrapper.

Reply:
👍